atlasdb-workload-server-api/src/main/java/com/palantir/atlasdb/workload/transaction/TransactionAction.java

```diff
@@ -21,6 +21,10 @@
 
 public interface TransactionAction {
 
+    /** Table to apply transaction to. */
```

Comment:
nit: the action to.

Also, is there a reason we went for strings as opposed to `TableReference`s?

Reply:
Yes, `TableReference` requires us to pull in a dependency on atlasdb-api, which I'm trying to avoid doing for the workload server. I _think_ that's the right approach, as that way this later on could let us migrate to a standalone service, which does not care for `namespace`.